app/mesh/1.6.x/index.md

```diff
@@ -34,11 +34,117 @@ every cloud and environment.
 </div>
 <br>
 <center>
-  <img src="/assets/images/docs/mesh/kong-mesh.png" width="500px"/>
-  <br>
   <i>{{site.mesh_product_name}} extends CNCF's Kuma and Envoy to provide an
   enterprise-grade service mesh with unique features in the service mesh
   landscape, while still relying on a neutral foundation.</i>
+
+<table class="mesh-features">
```

Comment:
```suggestion
<table class="mesh-features" style="width: fit-content">
```
If you want the table to be centered, then need to add this (or define the `mesh-features` class and add the styling to it.)